knip.ts

```diff
@@ -4,7 +4,7 @@ const config: KnipConfig = {
   entry: ["src/main.ts"],
   project: ["src/**/*.ts"],
   ignore: [],
-  ignoreBinaries: ["publish"],
+  ignoreBinaries: ["publish", "src/adapters/supabase/types/database.ts"],
```

Comment:
Are you sure this needs to be added to ignoreBinaries?

Reply:
Good catch, that was me experimenting with Knip to ignore the auto-generated code from Supabase, that was causing problems. The fix was to add it ignored inside the `.eslintrc`. Will fix